modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -5,9 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with applications and components
 
-The *Topology view* in the *Developer* perspective of the web console provides the following options to interact with applications and components:
+The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with applications and components:
 
-* Use the *Shortcuts* menu on the upper-right of the screen to navigate components in the *Topology view*.
+* Use the *Shortcuts* menu on the upper-right of the screen to navigate components in the *Graph view*.
```

Comment:
Hmmm... the Shortcuts menu is NEXT to the icon, it's not the name of the icon. it's a different interaction point.

Maybe something like:
```suggestion
* Use the icon (<insert the icon here>) in the upper-right of the screen to navigate components in the *Graph view*.
```